packages/pluggableWidgets/video-player-web/src/VideoPlayer.editorPreview.tsx

```diff
@@ -15,37 +14,25 @@ export class preview extends Component<VideoPlayerPreviewProps, {}> {
         return (
             <SizeContainer
                 className={classNames("video-player-container", this.props.class)}
-                style={{ ...parseStyle(this.props.style), minHeight: "400px" }}
+                style={{ ...parseStyle(this.props.style), minHeight: "400px" }} // TODO: remove the min height when aspect ratio height works with percentages inside SizeContainer
                 widthUnit={this.props.widthUnit}
                 width={this.props.width ?? 0}
                 heightUnit={this.props.heightUnit}
                 height={this.props.height ?? 0}
                 tabIndex={0}
             >
-                {this.renderPlayers()}
+                {this.props.showControls ? (
+                    <img
+                        className="widget-video-player-preview-image"
+                        src={videoWithControlsPreview}
+                        alt="Video with controls preview"
+                    />
+                ) : (
+                    <img className="widget-video-player-preview-image" src={videoPreview} alt="Video preview" />
```

Comment:
- Do we really need alt text in the preview mode of Studio? 
- Personal opinion, but I think the following is more readable as it better highlights the differences:
```
<img
  className="widget-video-player-preview-image"
  src={this.props.showControls ? videoWithControlsPreview : videoPreview}
  alt={this.props.showControls ? "Video with controls preview" : "Video preview"} // <- only if alt text is relevant
/>
```

Reply:
I changed it. I kept the alt text, because it can benefit PE accessibility in the future. In general, we also apply other accessibility attributes in design mode preview by reusing the React components located in the `src/components` folder. Besides that my IDE also gave a warning that it was missing.